sklearn/ensemble/_bagging.py

```diff
@@ -969,13 +1027,27 @@ def predict_log_proba(self, X):
             The training input samples. Sparse matrices are accepted only if
             they are supported by the base estimator.
 
+        **predict_params : dict
+            Parameters routed to the `predict_log_proba` or `predict_proba` method of
+            the sub-estimators via the metadata routing API.
```

Comment:
a comment about how things are routed to `predict`, `predict_proba`, or `predict_log_proba`